packages/syft/src/syft/core/tensor/autodp/gamma_tensor.py

```diff
@@ -2381,6 +2382,188 @@ def _squeeze(state: dict) -> jax.numpy.DeviceArray:
             sources=output_state,
         )
 
+    def any(
+        self,
+        axis: Optional[Union[int, Tuple[int, ...]]] = None,
+        keepdims: Optional[bool] = False,
+        where: Optional[ArrayLike] = None,
+    ) -> GammaTensor:
+        func = "any"
+
+        def _any(state: dict) -> jax.numpy.DeviceArray:
```

Comment:
Mild performance improvement: this method doesn't appear to have state (aside from the argument called "state"). Perhaps we could move it outside of the any() function so that we don't pay the cost of creating this method whenever we'd like to call any()?

Reply:
(ditto for other methods)

Reply:
if we dont need different objects, yeah definitely sounds like an improvement

Reply:
fix